agito.py

```diff
@@ -645,23 +645,50 @@ def substitute(m):
 
 	return message
 
+def use_timezone(timezone):
+    """A git filter-branch filter that converts author and commit times to specifiedtimezone"""
+
+    # import locally, the pytz and dateutil Python modules are not required normally
```

Comment:
I'd prefer these were at the top of the file. Since pytz appears to be a non-standard module, is it possible to do this without using pytz?